### ISSUE TITLE
F3 spoofer

### ISSUE THREAD

**Issue by l1ving** [kami-blue/client#1533]
Created by: Gravitywave `(308285665467039746)`

Hey, I would find it really cool if there was an option to modify some of the F3 lines like faking coordinates when streaming my screen or hiding the client in the top left. Thanks!

**Comment by l1ving** [comment]
Blocked by #790

**Comment by Xiaro** [comment]
Not really, you just need to mixin into the debug gui and remove some of the strings from it

**Comment by l1ving** [comment]
Yes, but it's blocked by #790 because I'd like a proper gui to edit it, similar to inertia

**Comment by ToxicAven** [comment]
This has been completed.
